### PR TITLE
Xie8899 8.0 update timesheet activity report

### PR DESCRIPTION
**Pull request by victor-ditay**

### CODE REVIEW

**Comment by victor-ditay** [comment]
To replace #109 Please have a check

Please have a check @noahzaozao @elicoidal @seb-elico

**Comment by elicoidal** [comment]
Travis shoudl be green

**Comment by victor-ditay** [comment]
> Travis should be green

@elicoidal Yes but better let be fix master branch by #154 and rebase this

**Comment by seb-elico** [comment]
@victormartinelicocorp I don't understand why you want to replace the original PR #109. The repo is public, so is the fork of XP. So you should be able to push to his branch so that it updates the original PR! And we won't lose all the previous comments. Any reason why?

**Review by elicoidal — APPROVED** [review]

File: project_task_category/README.rst
Comment (elicoidal) [inline project_task_category/README.rst:24]: remove

File: project_task_category/README.rst
Comment (elicoidal) [inline project_task_category/README.rst:41]: update

**Review by elicoidal — APPROVED** [review]
small details

**Comment by elicoidal** [comment]
@seb-elico we probably do not have access anymore to original repo

File: business_requirement_activity_report/README.rst
Comment (noahzaozao) [inline business_requirement_activity_report/README.rst:6]: why the name change to Business Requirement Activity Report , so it should be in the OCA business requirement repo ?

File: business_requirement_activity_report/report/timesheet_activity_report.py
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report.py:56]: be careful with this field , we did migration of elico 8 , I've commented this fields in many modules.

File: business_requirement_activity_report/report/timesheet_activity_report.py
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report.py:68]: not sure this field should be project.category or project.project.category

File: business_requirement_activity_report/report/timesheet_activity_report.py
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report.py:120]: be careful with this field , we commented this field during the migration

File: business_requirement_activity_report/report/timesheet_activity_report_view.xml
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report_view.xml:17]: be careful with this field , we commented this field during the migration

File: business_requirement_activity_report/report/timesheet_activity_report_view.xml
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report_view.xml:64]: be careful with this field , we commented this field during the migration

File: business_requirement_activity_report/report/timesheet_activity_report_view.xml
Comment (noahzaozao) [inline business_requirement_activity_report/report/timesheet_activity_report_view.xml:79]: be careful with this field , we commented this field during the migration

File: project_task_category/models/task_category.py
Comment (noahzaozao) [inline project_task_category/models/task_category.py:26]: be careful with this field , we commented this field during the migration , maybe it should be 'project.category' ?

File: business_requirement_activity_report/README.rst
Comment (martintorresvm) [reply business_requirement_activity_report/README.rst:6]: > To replace #109 Please have a check
Please have a check @noahzaozao @elicoidal @seb-elico

Not really idea I continue the left PR created @xie8899 to solve merge problems and not sure about this maybe @elicoidal could reply or you could have a look on previous history of comments.

File: business_requirement_activity_report/report/timesheet_activity_report.py
Comment (martintorresvm) [reply business_requirement_activity_report/report/timesheet_activity_report.py:56]: Yes I would use asap this field related to a new module we recently proposed to https://github.com/OCA/project/pull/225 or just drop it to merge this PR the sooner the better.

**Comment by seb-elico** [comment]
@noahzaozao thanks for the warning about `task_categ_id`. Indeed, we commented it out because right now, the field is in a tag. It should be solved very soon ( @victormartinelicocorp is working on it), so let's wait for that before we merge this PR.

File: business_requirement_activity_report/README.rst
Comment (seb-elico) [reply business_requirement_activity_report/README.rst:6]: @noahzaozao this module is very specific, it depends on way too many modules to be included in the OCA BR repo. Definitely belongs to odoo-addons

**Comment by elicoidal** [comment]
@seb-elico @victormartinelicocorp Let's move this topic asap now

**Comment by seb-elico** [comment]
@victormartinelicocorp can you make sure Travis is building please? XML files don't seem to be formatted very well

**Comment by elicoidal** [comment]
will be replaced by new API clean version

**Pull request closed without merging**
